vendor/cloud.google.com/go/storage/bucket.go

```diff
@@ -1991,9 +1991,7 @@ func (it *ObjectIterator) fetch(pageSize int, pageToken string) (string, error)
 	if it.bucket.userProject != "" {
 		req.UserProject(it.bucket.userProject)
 	}
-	if pageSize > 0 {
-		req.MaxResults(int64(pageSize))
-	}
+	req.MaxResults(5000)
```

Comment:
Please add a comment here that this is done because the default is 1000 and we need to find a way to make this configurable

Reply:
done.

Reply:
Where is the comment?

Reply:
No the files will not get updated here.
please check: [link](https://github.com/GoogleCloudPlatform/gcsfuse/pull/727/files)